src/Tribe/Repositories/Traits/Post_Attendees.php

```diff
@@ -0,0 +1,271 @@
+		$user_ids = implode( ', ', $user_ids );
+
+		$alias_event = 'ticket_attendee_event';
+		$alias_user  = 'ticket_attendee_user';
+
+		$event_meta_keys = method_exists( $this, 'attendee_to_event_keys' ) ? $this->attendee_to_event_keys() : [];
+		$event_meta_keys = array_map( [ $wpdb, '_real_escape' ], $event_meta_keys );
+		$event_meta_keys = '"' . implode( '", "', $event_meta_keys ) . '"';
+
+		$user_meta_key = method_exists( $this, 'attendee_to_user_key' ) ? $this->attendee_to_user_key() : 'null';
+		$user_meta_key = $wpdb->_real_escape( $user_meta_key );
+
+		// Join to the meta that relates attendees to events.
+		$repo->filter_query->join( "
+				LEFT JOIN `{$wpdb->postmeta}` AS `{$alias_event}`
+					ON `{$alias_event}`.`meta_value` = `{$wpdb->posts}`.`ID`
+			", $alias_event );
+
+		// Join to the meta that relates users to attendees.
+		$repo->filter_query->join( "
+				LEFT JOIN `{$wpdb->postmeta}` AS `{$alias_user}`
+					ON `{$alias_user}`.`post_id` = `{$alias_event}`.`post_id`
+			", $alias_user );
+
+		$repo->where_clause( "
+				`{$alias_event}`.`meta_key` IN ( {$event_meta_keys} )
+				AND `{$alias_user}`.`meta_key` = '{$user_meta_key}'
+				AND `{$alias_user}`.`meta_value` IN ( {$user_ids} )
+			" );
+	}
+
+	/**
+	 * Filters events to include only those that do not match the provided attendee(s).
+	 *
+	 * @since TBD
+	 *
+	 * @param int|array $user_ids The user ID(s) to filter out.
+	 */
+	public function filter_by_attendee_user_not_in( $user_ids ) {
+		$repo = $this;
+
+		// If the repo is decorated, use that.
+		if ( ! empty( $this->decorated ) ) {
+			$repo = $this->decorated;
+		}
+
+		global $wpdb;
+
+		$user_ids = (array) $user_ids;
+		$user_ids = array_map( 'absint', $user_ids );
+		$user_ids = array_unique( $user_ids );
+		$user_ids = implode( ', ', $user_ids );
+
+		$alias_event = 'sub_ticket_attendee_event';
+		$alias_user  = 'sub_ticket_attendee_user';
+
+		$event_meta_keys = method_exists( $this, 'attendee_to_event_keys' ) ? $this->attendee_to_event_keys() : [];
+		$event_meta_keys = array_map( [ $wpdb, '_real_escape' ], $event_meta_keys );
+		$event_meta_keys = '"' . implode( '", "', $event_meta_keys ) . '"';
+
+		$user_meta_key = method_exists( $this, 'attendee_to_user_key' ) ? $this->attendee_to_user_key() : 'null';
+		$user_meta_key = $wpdb->_real_escape( $user_meta_key );
+
+		$repo->where_clause( "
+				NOT EXISTS (
+					SELECT 1
+					FROM
+						`{$wpdb->postmeta}` AS `{$alias_event}`
+					WHERE
+						`{$alias_event}`.`meta_value` = `{$wpdb->posts}`.`ID`
+						AND `{$alias_event}`.`meta_key` IN ( {$event_meta_keys} )
+						AND	NOT EXISTS (
+							SELECT 1
+							FROM
+								`{$wpdb->postmeta}` AS `{$alias_user}`
+							WHERE
+								`{$alias_user}`.`post_id` = `{$alias_event}`.`post_id`
+								AND `{$alias_user}`.`meta_key` = '{$user_meta_key}'
+								AND `{$alias_user}`.`meta_value` IN ( {$user_ids} )
+							LIMIT 1
+						)
+					LIMIT 1
+				)
+			" );
+	}
```

Comment:
:no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: de4fb8fc8edb68a864e302dca4c33ee2</sub>